terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,30 @@
+environment = "cidev"
+aws_profile = "development-eu-west-2"
+
+account_web_url = "https://account.cidev.aws.chdev.org"
+api_url = "https://api.cidev.aws.chdev.org"
+account_validator_max_file_size = "300MB"
+allowed_company_prefixes = "SC,NI,OC,SO,R,AP"
+
+cache_server = "redis"
+cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
+
+chs_url = "https://cidev.aws.chdev.org"
+
+cookie_domain = "cidev.aws.chdev.org"
+cookie_name = "__SID"
+cookie_secret = "ChGovUk-XQrbf3sLj2abFxIY2TlapsJ"
```

Comment:
This is a secret and should be in the vault

Reply:
Removed from both, dev and staging.